computer-vision/README.md

```diff
@@ -0,0 +1,42 @@
+# Computer Vision
+
+The repository shall focus on the traditional aspects of computer vision, particularly multiview geometry. 
+
+## Pre - Requisites
+1. It is expected that one is familiar with some of the concepts of Linear Algebra. No course can beat the one provided by [Gilbert Strang](https://www.youtube.com/playlist?list=PLE7DDD91010BC51F8). Make sure that one sticks patiently as some of the concepts might seem daunting initially. For gaining some motivation and intuition for some of the concepts, one can refer to the [Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab).
+2. For implementation of some of the concepts, [OpenCV](https://opencv.org/releases/) will help you. Note that you are required to have some background in python. There are many tutorials available, such as [the one by Corey Schafer](https://www.youtube.com/watch?v=YYXdXT2l-Gg&list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU), or [by sentdex](https://www.youtube.com/playlist?list=PLQVvvaa0QuDe8XSftW-RAxdo6OmaeL85M). 
+
+## Basic Roadmap
+
+1. To learn basic concepts that are used in image processing and computer vision, [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) is a good place to start. The course starts with introducing various image processing techniques required, before moving on to introducing camera geometry, such as the pinhole model, intrinsic and extrinsic camera parameters and analysing the point correspondences between two views of the same scene. The course moves on to introduce the concept of features, scale invariance and then introducing the state of the art method, Scale Invariant Feature Transform (SIFT). One can complete the whole course in order to get an idea on different computer vision and image processing techniques. 
```

Comment:
Introduction to Computer Vision by udacity has a refresher for Image Processing but hasn't covered everything. Don't recommend this course for Image Processing. This course is mainly focused on CV. This should be put into the main course section.